seahorn/jobs/array_list_comparator_string/aws_array_list_comparator_string_harness.klee.c

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ */
+
+#include <seahorn/seahorn.h>
+#include <aws/common/array_list.h>
+#include <array_list_helper.h>
+#include <string_helper.h>
+#include <utils.h>
+
+void sea_dsa_alias(const void *p, const void *q){
```

Comment:
this has been removed.

In general, no `sea_dsa_alias` is a special function known by seahorn. For klee, a dummy implementation is good. But then this file is not compatible with seahorn.